### PR TITLE
Change host property name to accept 'marklogic.host'

### PR DESCRIPTION
**Pull request by sastafford**
See #58.  Not quite done with this ticket, but just wanted to illustrate how I would implement.  Tested locally.

### CODE REVIEW

File: src/main/groovy/com/rjrudin/marklogic/gradle/MarkLogicPlugin.groovy [hunk 1]
```diff
@@ -426,4 +422,8 @@ class MarkLogicPlugin implements Plugin<Project> {
         deployer.setCommands(commands)
         return deployer
     }
+    
+    def host(project) {
```
Comment (rjrudin) [inline src/main/groovy/com/rjrudin/marklogic/gradle/MarkLogicPlugin.groovy]: Can you name this "getHost"?

File: src/main/groovy/com/rjrudin/marklogic/gradle/MarkLogicPlugin.groovy [hunk 2]
```diff
@@ -426,4 +422,8 @@ class MarkLogicPlugin implements Plugin<Project> {
         deployer.setCommands(commands)
         return deployer
     }
+    
+    def host(project) { 
+      project.hasProperty("mlHost") ? project.property("mlHost") : project.property("marklogic.host")
```
Comment (rjrudin) [inline src/main/groovy/com/rjrudin/marklogic/gradle/MarkLogicPlugin.groovy]: Test out what happens if neither mlHost nor marklogic.host is present - project.property may throw an error if the property doesn't exist.

For testing, I have ml-gradle and the gh-pages branch of marklogic-java in the same "workspace" directory. I change ml-gradle, then I do something like "gradle -Pversion=something publish". Then I go to examples/sample-project and modify its gradle.properties file to depend on version "something" of ml-gradle. Then to test this, you could just run "gradle -i tasks" to see what ml-gradle prints out for the property values.

**Comment by sastafford** [comment]
Feel free to revert the merge if there is anything wrong.  I think I merged without the proper review

**Comment by rjrudin** [comment]
It'd be better to have a separate example project that shows Spring properties being used - I'm going to revert sample-project back to using the normal properties, as that's my main example project for getting started.

**Comment by rjrudin** [comment]
Hey Scott, I reverted all of it because MarkLogicPlugin will throw errors in case neither the default property nor the Spring property exists. The code for each property needs to be written as (pseudocode):

def getUsername
  if (project.hasProperty("mlUsername")) then project.property("mlUsername")
  else if (project.hasProperty("marklogic.username") then project.property("marklogic.username")
  
Those hasProperty checks are needed because Gradle will throw an error if project.property is called and the property doesn't exist.

**Pull request merged**